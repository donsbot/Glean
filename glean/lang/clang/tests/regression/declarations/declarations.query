query: |
    R where cxx1.DeclarationSrcRange { _, R }
transform: [gensort, []]
