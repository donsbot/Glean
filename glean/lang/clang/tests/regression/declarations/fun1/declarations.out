[
  "@generated",
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 9,
      "columnBegin": 1,
      "lineEnd": 9,
      "columnEnd": 9
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 10,
      "columnBegin": 1,
      "lineEnd": 10,
      "columnEnd": 22
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 10,
      "columnBegin": 9,
      "lineEnd": 10,
      "columnEnd": 14
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 10,
      "columnBegin": 17,
      "lineEnd": 10,
      "columnEnd": 21
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 11,
      "columnBegin": 1,
      "lineEnd": 11,
      "columnEnd": 24
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 11,
      "columnBegin": 9,
      "lineEnd": 11,
      "columnEnd": 11
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 11,
      "columnBegin": 14,
      "lineEnd": 11,
      "columnEnd": 17
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 11,
      "columnBegin": 20,
      "lineEnd": 11,
      "columnEnd": 23
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 12,
      "columnBegin": 1,
      "lineEnd": 12,
      "columnEnd": 8
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 13,
      "columnBegin": 1,
      "lineEnd": 13,
      "columnEnd": 16
    }
  }
]